qa/L0_perf_resnet/run_test.sh

```diff
@@ -43,95 +42,77 @@ export CUDA_VISIBLE_DEVICES=0
 
 RET=0
 
-for STATIC_BATCH in $STATIC_BATCH_SIZES; do
-    for DYNAMIC_BATCH in $DYNAMIC_BATCH_SIZES; do
-        for INSTANCE_CNT in $INSTANCE_COUNTS; do
-            if (( ($DYNAMIC_BATCH > 1) && ($STATIC_BATCH >= $DYNAMIC_BATCH) )); then
-                continue
-            fi
-
-            MAX_BATCH=${STATIC_BATCH} && \
-                (( $DYNAMIC_BATCH > $STATIC_BATCH )) && \
-                MAX_BATCH=${DYNAMIC_BATCH}
-            CONCURRENCY=$(( $INSTANCE_CNT * $DYNAMIC_BATCH * 2 ))
-            if (( $CONCURRENCY < $REQUIRED_CONCURRENCY )); then
-                CONCURRENCY=$REQUIRED_CONCURRENCY
-            fi
-
-            if (( $DYNAMIC_BATCH > 1 )); then
-                NAME=${MODEL_NAME}_sbatch${STATIC_BATCH}_dbatch${DYNAMIC_BATCH}_instance${INSTANCE_CNT}
-            else
-                NAME=${MODEL_NAME}_sbatch${STATIC_BATCH}_instance${INSTANCE_CNT}
-            fi
-
-            rm -fr models && mkdir -p models && \
-                cp -r $MODEL_PATH models/. && \
-                (cd models/$MODEL_NAME && \
-                        sed -i "s/^max_batch_size:.*/max_batch_size: ${MAX_BATCH}/" config.pbtxt && \
-                        echo "instance_group [ { count: ${INSTANCE_CNT} }]" >> config.pbtxt)
-            if (( $DYNAMIC_BATCH > 1 )); then
-                (cd models/$MODEL_NAME && \
-                        echo "dynamic_batching { preferred_batch_size: [ ${DYNAMIC_BATCH} ] }" >> config.pbtxt)
```

Comment:
Why are we removing the dynamic batching benchmarks from L0_perf_resnet?

Reply:
It didn't test exactly what we wanted to test. After discussing with @deadeyegoodwin we decided to remove it from this test and add another test later on that tests dynamic batching performance more accurately.

Reply:
Can you elaborate on how it doesn't test what we are interested in? Do we want to test a larger set of preferred_batch_size to ascertain the performance of our dynamic_batcher?

Reply:
I think David can better answer that question. We can discuss this on slack further.